spec/pushmi_pullyu/swift_depositer_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
 RSpec.describe PushmiPullyu::SwiftDepositer do
-  it 'not valid with nil arguments' do
```

Comment:
Oops you merged the wrong two tests. This one ( 'not valid with nil arguments') was completely fine. It was the other two tests in this file. They can be merged as they are testing the exact same pieces of code. Hence you can merge them together and have less code as a result (including one less VCR file)